src/TornadoWebBehavior.ts

```diff
@@ -69,6 +69,15 @@ export class TornadoWebBehavior implements IInteractiveBehavior {
                 event.ctrlKey);
         });
 
+        this.columns.on("keypress", (event : KeyboardEvent, dataPoint: TornadoChartPoint) => {
+            if(event?.key == "Enter")
+            {
+                selectionHandler.handleSelection(
+                    dataPoint,
+                    event.shiftKey);
```

Comment:
multiselection not working with Ctrl key

Reply:
done